tests/tensorflow/nightly/vision-retinanet.libsonnet

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local experimental = import '../experimental.libsonnet';
+local common = import 'common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local timeouts = import 'templates/timeouts.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+local utils = import 'templates/utils.libsonnet';
+
+{
+  local retinanet = common.TfVisionTest + {
+    modelName: 'vision-retinanet',
+    scriptConfig+: {
+      experiment: 'retinanet_resnetfpn_coco',
+    },
+    PreemptibleTpu:: {
+      tpuSettings+: {
+        preemptible: false,
+      },
+    },
+  },
+  local functional = common.Functional {
+    scriptConfig+: {
+      paramsOverride+: {
+        task: {
```

Comment:
Move these common fields to `retinanet` instead:

```
      paramsOverride+: {
        task: {
          annotation_file: '$(COCO_DIR)/instances_val2017.json',
	  train_data: {
            input_path: '$(COCO_DIR)/train*',
          },
	  validation_data: {
            input_path: '$(COCO_DIR)/val*',
          },
        },
```

The input data is common to both `functional` and `convergence`, but right now the convergence test doesn't point to any input data.